### PR TITLE
Part Item Type: add more redirects

### PR DESCRIPTION
**Pull request by davidrans**
Connects https://github.com/iFixit/react-commerce/issues/529

### CODE REVIEW

**Review by masonmcelvain — DISMISSED** [review]
CR ✌🏻

**Comment by erinemay** [comment]
dev_block 🧷 @davidrans can you add `Optical_Drives` and `Power_Supplies`? https://github.com/iFixit/react-commerce/issues/529#issuecomment-1202940138

**Comment by davidrans** [comment]
It looks to me like `Power_Supplies` and `Optical_Drives` are still legit item types:
https://www.ifixit.com/Parts/Xbox_One_S/Power_Supplies
https://www.ifixit.com/Parts/PlayStation_4_Slim/Optical_Drives

File: frontend/legacyPartCollectionRoutes.js [hunk 1]
```diff
@@ -66,11 +68,13 @@ const legacyTagsWithNoItemType = [
    'PC_Cards',
    'PCIe',
    'Propellers',
+   'Prying_and_Opening',
    'Pulleys',
    'Pumps',
    'Retail_Display',
    'RJ-11_Boards',
    'SD_Card_Slots',
+   'Sensors',
```
Comment (davidrans) [inline frontend/legacyPartCollectionRoutes.js:77]: Looks like this is actually valid, and we don't want a comprehensive redirect: https://www.ifixit.com/Parts/Samsung_Galaxy_S9/Sensors

```suggestion
```

**Comment by davidrans** [comment]
un_dev_block :+1:

**Review by sterlinghirsh — APPROVED** [review]

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by erinemay** [comment]
Dang, ok this is maybe a bigger/unrelated issue then. Or maybe a data issue? Why are only some collections mapping the same item type products to different item types?

https://www.ifixit.com/Store/Game-Console/Xbox-One/Power_Supplies redirects to https://www.ifixit.com/Parts/Xbox_One/Power_Supplies which exists and contains the same products as the legacy collection.

https://www.ifixit.com/Store/Game-Console/Steam-Deck/Power_Supplies redirects to https://www.ifixit.com/Parts/Steam_Deck/Power_Supplies which doesn't exist (no Power_Supplies item type in this collection). And all the products that were under that legacy collection are under https://www.ifixit.com/Parts/Steam_Deck/Power_Adapters. Is this a specialized redirect?

**Comment by erinemay** [comment]
QA 🌡️ Those three examples in the issue corresponding to these 3 changes now redirect appropriately.

**Comment by erinemay** [comment]
> Dang, ok this is maybe a bigger/unrelated issue then. Or maybe a data issue? Why are only some collections mapping the same item type products to different item types?
> 
> https://www.ifixit.com/Store/Game-Console/Xbox-One/Power_Supplies redirects to https://www.ifixit.com/Parts/Xbox_One/Power_Supplies which exists and contains the same products as the legacy collection.
> 
> https://www.ifixit.com/Store/Game-Console/Steam-Deck/Power_Supplies redirects to https://www.ifixit.com/Parts/Steam_Deck/Power_Supplies which doesn't exist (no Power_Supplies item type in this collection). And all the products that were under that legacy collection are under https://www.ifixit.com/Parts/Steam_Deck/Power_Adapters. Is this a specialized redirect?

Moved this into the issue. Can deal with separately.

**Pull request merged**